workers/tasks/export/notes.ts

```diff
@@ -88,10 +87,6 @@ export const getPandocNotesByHash = (notesData: NotesData): PandocNotes => {
 	return index;
 };
 
-export const getCslJsonForPandocNotes = (notes: PandocNotes) => {
```

Comment:
I just inlined this at its only callsite because it felt like it wasn't really pulling its weight as a standalone fn.